Sources/SQL/Core/main.swift

```diff
@@ -0,0 +1,141 @@
+extension Table {
+    public static func f(field: Field) -> DeclaredField {
```

Comment:
Is this a temporary function? If not, I don't think it's good convention to have one-letter function names.